src/lib/components/menu/Menu.svelte

```diff
@@ -3,11 +3,15 @@
   const globalActiveMenu = writable(undefined);
 </script>
 
-<script>
-  import { createEventDispatcher, setContext, onMount } from "svelte";
+<script lang="ts">
+  import { guidGenerator } from "$lib/util/guid";
+  import { createEventDispatcher, onMount, setContext } from "svelte";
   import { writable } from "svelte/store";
   import { fade } from "svelte/transition";
-  import { guidGenerator } from "$lib/util/guid";
+
+  export let dark = false;
+  setContext("rill:menu:dark", dark);
```

Comment:
note for the future:
might be good for us to set this to `"rill:theme:dark"`. In theory, we could set a dark theme at the top level of the app and have it propagate down later